week-3/D-array-filter/exercise.js

```diff
@@ -6,8 +6,26 @@
   - Do not edit any of the other existing code
 */
 
+function myFilter(item)
+{
+  if (item === null) {
+    return false;
+  } else if (item === false) {
+    return false;
+  } else if (typeof item === "string") {
+    return false;
+  } else if (item.length < 2 ){
+    return false ;
+  }
+    else {
+      return true;
+}
```

Comment:
:+1: You don't need to use `else`  when the `if` statement contains a `return`, as the function will end at that point.

Reply:
Ok thanks Luke